### ISSUE TITLE
关于OCR屏幕选区不准确的问题

### ISSUE THREAD

**Issue by DQ-HollyLee** [SoftBlack42/StarPie#79]
<img width="2559" height="1439" alt="Image" src="https://github.com/user-attachments/assets/21b5f60f-8449-4528-aa34-2077ed1b0461" />
<img width="2559" height="1439" alt="Image" src="https://github.com/user-attachments/assets/aa1870f0-99df-467c-8430-67d8b8601c03" />
OCR选择屏幕区域时，实际的区域要比框选的区域偏左上一些，屏幕分辨率为2560x1440，开启HDR
因为Windows本地OCR识别率偏低，故使用deepseek-v4-flash-vision-exp模型测试

**Comment by DQ-HollyLee** [comment]
版本为v1.7.2-beta.2

**Comment by SoftBlack42** [comment]
该功能属于测试阶段，之前已发现这样的问题，在下一阶段就会修👌

**Comment by Foundnova** [comment]
感谢您的issue，该情况的异常问题，已经彻底解决，请期待新版本
详情视频链接：
https://github.com/user-attachments/assets/a9742ce5-bc1d-4655-b885-4cee6a34b2f8
